src/main/java/chess/controller/BoardController.java

```diff
@@ -0,0 +1,55 @@
+package chess.controller;
+
+import chess.Dto.BoardDto;
+import chess.Dto.MoveRequest;
+import chess.Dto.PathDto;
+import chess.domain.piece.PieceColor;
+import chess.service.ChessService;
+import java.util.List;
+import java.util.Map;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/board")
+public class BoardController {
+
+    private final ChessService service;
+
+    public BoardController(ChessService service) {
+        this.service = service;
+    }
+
+    @GetMapping(value = {"","/restart"})
+    public BoardDto getNewBoard() {
+        return new BoardDto(service.restartBoard());
+    }
+
+    @GetMapping("/status")
+    public boolean isEnd(){
```

Comment:
```suggestion
    public boolean isEnd() {
```

컨벤션을 위반하셨습니다!
IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.